config.json

```diff
@@ -1,264 +1,1802 @@
     {
-      "resource_cost": 3, 
+      "cities": 6, 
       "cost": 38, 
-      "resource": "oil", 
-      "cities": 6
+      "resource_cost": 3, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 5, 
       "cost": 30, 
-      "resource": "oil", 
-      "cities": 5
+      "resource_cost": 2, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 4, 
       "cost": 23, 
-      "resource": "oil", 
-      "cities": 4
+      "resource_cost": 2, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 3, 
       "cost": 18, 
-      "resource": "oil", 
-      "cities": 3
+      "resource_cost": 2, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 2, 
       "cost": 10, 
-      "resource": "oil", 
-      "cities": 2
+      "resource_cost": 2, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "dark"
     }, 
     {
-      "resource_cost": 1, 
+      "cities": 1, 
       "cost": 6, 
-      "resource": "oil", 
-      "cities": 1
+      "resource_cost": 1, 
+      "resources": [
+        "oil"
+      ], 
+      "type": "dark"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 7, 
       "cost": 50, 
-      "resource": "nuclear", 
-      "cities": 7
+      "resource_cost": 2, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 6, 
       "cost": 37, 
-      "resource": "nuclear", 
-      "cities": 6
+      "resource_cost": 2, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 5, 
       "cost": 32, 
-      "resource": "nuclear", 
-      "cities": 5
+      "resource_cost": 2, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 1, 
+      "cities": 3, 
       "cost": 21, 
-      "resource": "nuclear", 
-      "cities": 3
+      "resource_cost": 1, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 1, 
+      "cities": 2, 
       "cost": 13, 
-      "resource": "nuclear", 
-      "cities": 2
+      "resource_cost": 1, 
+      "resources": [
+        "nuclear"
+      ], 
+      "type": "dark"
     }, 
     {
-      "resource_cost": 2, 
+      "cities": 5, 
       "cost": 35, 
-      "resource": "oil&gas", 
-      "cities": 5
+      "resource_cost": 2, 
+      "resources": [
+        "oil", 
+        "gas"
+      ], 
+      "type": "light"
     }, 
     {
-      "resource_cost": 3, 
+      "cities": 5, 
       "cost": 22, 
-      "resource": "oil&gas", 
-      "cities": 5
+      "resource_cost": 3, 
+      "resources": [
```

Comment:
Made resources a list, so that way we don't have to do any string parsing to look for an '&'.  Now to see if a resource is valid to use on this power plant, we can just use a simple `if resource in card.resources`